### PR TITLE
Add Lambdas/constants.js to responsibilities of technical writers

### PR DESCRIPTION
**Pull request by aerfio**
**Description**

Changes proposed in this pull request:

- as in title, to prevent anyone from editing this file without their approval

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -15,6 +15,7 @@
 
 # Owners of all .md files in the repository
 *.md @kazydek @klaudiagrz @tomekpapiernik @bszwarc @mmitoraj @majakurcius @alexandra-simeonova
+/core-ui/src/components/Lambass/constants.js @kazydek @klaudiagrz @tomekpapiernik @bszwarc @mmitoraj @majakurcius @alexandra-simeonova
```
Comment (magicmatatjahu) [inline CODEOWNERS:18]: Lambdas :P

Comment (aerfio) [inline CODEOWNERS:18]:
```suggestion
/core-ui/src/components/Lambas/constants.js @kazydek @klaudiagrz @tomekpapiernik @bszwarc @mmitoraj @majakurcius @alexandra-simeonova
```

File: CODEOWNERS [hunk 2]
```diff
@@ -15,6 +15,7 @@
 
 # Owners of all .md files in the repository
 *.md @kazydek @klaudiagrz @tomekpapiernik @bszwarc @mmitoraj @majakurcius @alexandra-simeonova
+/core-ui/src/components/Lambas/constants.js @kazydek @klaudiagrz @tomekpapiernik @bszwarc @mmitoraj @majakurcius @alexandra-simeonova
```
Comment (bszwarc) [inline CODEOWNERS:18]: now it's Lambas 🤔

Comment (aerfio) [inline CODEOWNERS:18]:
```suggestion
/core-ui/src/components/Lambdas/constants.js @kazydek @klaudiagrz @tomekpapiernik @bszwarc @mmitoraj @majakurcius @alexandra-simeonova
```

**Pull request merged**